go/internal/tinder/driver_localdiscovery.go

```diff
@@ -239,19 +250,19 @@ func (ld *LocalDiscovery) sendRecordsToProximityPeers(ctx context.Context, recor
 	wg := sync.WaitGroup{}
 
 	for _, c := range conns {
-		if !manet.IsPrivateAddr(c.RemoteMultiaddr()) && !isProximityProtocol(c.RemoteMultiaddr()) {
-			continue
-		}
-
-		wg.Add(1)
-		go func(c network.Conn) {
-			if err := ld.sendRecordsTo(ctx, c.RemotePeer(), records); err != nil {
-				ld.logger.Error("unable to send records to newly connected peer",
-					zap.Error(err), zap.Stringer("peer", c.RemotePeer()))
-			}
+		if manet.IsPrivateAddr(c.RemoteMultiaddr()) || isProximityProtocol(c.RemoteMultiaddr()) {
+			wg.Add(1)
+			go func(c network.Conn) {
+				ld.logger.Debug("sending records to peer", zap.Stringer("peer", c.RemotePeer()))
+
+				if err := ld.sendRecordsTo(ctx, c.RemotePeer(), records); err != nil {
```

Comment:
Can you put a `TODO` comment above this line about checking if we only send record of the contact request rendezvous and we filter out the groups rendezvous?

Reply:
i've check and that the case: https://github.com/berty/berty/blob/11ede71bcb641a32edbfbf51de081c2213ab353b/go/internal/initutil/ipfs.go#L698